package.json

```
@@ -0,0 +1,26 @@
+{
+  "name": "nano_press",
+  "version": "1.0.0",
+  "description": "Nano Press automates the full lifecycle of a self-hosted Frappe/ERPNext deployment—from connecting a fresh server to shipping production-ready sites. It wraps Ansible automation, server health checks, app management, and site provisioning into a guided workflow that lives inside your Frappe desk.",
+  "main": "index.js",
+  "scripts": {
+		"postinstall": "yarn tailwind:build",
```

```suggestion
		"postinstall": "yarn tailwind:build",
```

This can be removed.

---

Done